internal/xds/translator/listener.go

```diff
@@ -467,6 +499,19 @@ func (t *Translator) addHCMToXDSListener(xdsListener *listenerv3.Listener, irLis
 	return nil
 }
 
+func routeConfigName(irListener *ir.HTTPListener) string {
+	// TODO(zhaohuabing): change the routeConfig name for HTTP listeners because they are merged into one route config
+	return irListener.Name
+}
+
+func httpListenerFilterChainName(irListener *ir.HTTPListener) string {
+	return irListener.Name
+}
+
+func tcpListenerFilterChainName(irRoute *ir.TCPRoute) string {
+	return irRoute.Name
```

Comment:
The current filter chain name for tls listener is irRoute name, should we also change it to the ir listener name to make it consistent with the https listener filter chain name?